md_template/model.md

```diff
@@ -0,0 +1,51 @@
+This is a simple template for setting up automatic report generation with GPkit.
+To generate this file, run
+```
+make model.pdf
+```
+in this directory.
+
+\section{Example Model}
+
+This example model minimizes $x + 1/x$ subject to the constraint
+$$x \ge x_{min}.$$
+The description above is hard coded as opposed to being derived from the Python
+code. This is a simple and effective way to document model equations.
+
+The full GP, derived from the Python implementation of the model, is
+```python
+#inPDF: replace with gp.generated.tex
+from simple_gp import SimpleGP
+m = SimpleGP()
+with open("gp.generated.tex", "w") as f:
+    f.write(m._repr_latex_())
+# TODO wish model name didn't get printed in the above
+```
+
+Sweeping over $x_{min}$ shows the relationship between $x$ and $x_{min}$.
+```python
+#inPDF: skip
+import numpy as np
+import matplotlib.pyplot as plt
+xmin = np.linspace(0.01, 3, 30)
+# TODO changing 0.01 to 0 above results in an error -- suspected GPkit bug
+m.substitutions.update({"x_{min}": ("sweep", xmin)})
+sol = m.solve(verbosity=0)
+plt.plot(sol["variables"]["x_{min}"], sol["cost"])
+plt.ylim((0, plt.ylim()[1]))
+plt.xlabel("$x_{min}$")
+plt.ylabel("cost")
+plt.savefig("simple_sweep.pdf")
+```
+```python
+#inPDF: skip
+# TODO: argh, the below doesn't work
```

Comment:
Yes, probably, that should be changed. hmm, also, we could add a flag for markdown (the image tag) that flips to latex, so that the markdown version could use an svg...